tests/test_generate_multiple_forecasts.py

```diff
@@ -0,0 +1,37 @@
+import os
+import pandas as pd
+import subprocess
+import random
+
+
+def test_generate_forecasts():
+    # Generate 100 random PV IDs between 1 and 50000
+    pv_ids = [random.randint(1, 50000) for _ in range(100)]
+
+    # Define the command to run your script
+    command = "python scripts/generate_multiple_forecasts.py " + " ".join(
+        map(str, pv_ids)
+    )
+
+    # Run your script
+    subprocess.run(command, shell=True)
+
+    # Check if the output file is created
+    assert os.path.exists(
+        "quartz_solar_forecast/dataset/forecasts.csv"
+    ), "forecast.csv doesn't exist"
+
+    # Load the output file
+    df = pd.read_csv("quartz_solar_forecast/dataset/forecasts.csv")
```

Comment:
Could you add some screen shots to the PR of this test passing? The current forecasts.csv doesn't exist in this location.